### ISSUE TITLE
Unable to train

### ISSUE THREAD

**Issue by jinfagang** [lxtGH/SFSegNets#36]
I got Nan loss:

```
04-08 10:51:43.489 NaN or Inf found in input tensor.
04-08 10:51:43.559 [epoch 28], [iter 8136 / 13576], [train main loss nan], [lr 0.001678]
04-08 10:51:43.560 NaN or Inf found in input tensor.
04-08 10:51:43.632 [epoch 28], [iter 8137 / 13576], [train main loss nan], [lr 0.001678]
04-08 10:51:43.632 NaN or Inf found in input tensor.
04-08 10:51:43.965 [epoch 28], [iter 8138 / 13576], [train main loss nan], [lr 0.001678]
04-08 10:51:43.965 NaN or Inf found in input tensor.
04-08 10:51:44.036 [epoch 28], [iter 8139 / 13576], [train main loss nan], [lr 0.001678]
04-08 10:51:44.037 NaN or Inf found in input tensor.

```

**Comment by lxtGH** [comment]
![image](https://user-images.githubusercontent.com/17202268/113992029-9f8fb600-9885-11eb-9e5c-f9baf99ed9c2.png)
I do not meet such error.
I try by myself. It seems normal in the first epoch.

**Comment by jinfagang** [comment]
@lxtGH 1 epoch is converged. but from epoch 2, loss become bigger and bigger:

![image](https://user-images.githubusercontent.com/21303438/113999166-5e4ed480-988c-11eb-9e68-4ca1c3764405.png)

**Comment by lxtGH** [comment]
What about using pt.1.4?

**Comment by jinfagang** [comment]
@lxtGH Dude, to verify your code logic, I reinstall whole enviroment including pytorch 1.4. Same behavior. 
You can test it on pytorch 1.8 ....  All people from mmseg can not reproduce your result...

**Comment by lxtGH** [comment]
@jinfagang What is your training setting using pt1.4 ?

**Comment by lxtGH** [comment]
Also. what dataset you use ? Cityscapes or your custom datasets?

**Comment by jinfagang** [comment]
@lxtGH I am training on custom datasets. I am just following exactly same steps like cityspaces. However I think the biggest issue is that I also tried cityscapes same problem.

**Comment by lxtGH** [comment]
In summary, using pt.1.4 on cityscapes still have this problem?

**Comment by jinfagang** [comment]
@lxtGH Yes

**Comment by jinfagang** [comment]
@lxtGH Besides. I double checked pytorch 1.7. You can try pytorch 1.7, this should not be pytorch version issue. there are some place went wrong. Try debug with pytorch 1.7 to see what caused the issue.

**Comment by lxtGH** [comment]
@jinfagang OK I will re-train the model using pt1.4. I will also train the code using pt.1.7 using our lab server but the time maybe longer. Keep tune it.

**Comment by lxtGH** [comment]
@jinfagang Another thing I want to ask, which config you use for training cityscape ?

**Comment by jinfagang** [comment]
@lxtGH  I didn't made any modifications. I just using the sfresnet18 model without dsn since dsn has bug.

**Comment by lxtGH** [comment]
Hi! Guys. I did not meet any error,. The results are normal using pt.1.6 and pt.1.4 
![image](https://user-images.githubusercontent.com/17202268/115104613-3fd39200-9f8c-11eb-996d-f70a337108ba.png)
